src/main/kotlin/net/atos/zac/mail/MailService.kt

```diff
@@ -82,11 +86,37 @@ class MailService @Inject constructor(
         private const val MAIL_BIJLAGE = "Bijlage"
         private const val MAIL_ONDERWERP = "Onderwerp"
         private const val MAIL_BERICHT = "Bericht"
+
+        private const val MAIL_SMTP_USER = "mail.smtp.user"
+        private const val MAIL_SMTP_AUTH = "mail.smtp.auth"
     }
 
     val gemeenteMailAdres
         get() = MailAdres(configuratieService.readGemeenteMail(), configuratieService.readGemeenteNaam())
 
+    @Synchronized
```

Comment:
Could we annotate this method with @PostConstruct to just run it once, and avoid synchronization issues?